app/resources/forms.js.template

```diff
@@ -376,13 +376,13 @@ function make_form_expandable(expandedSection) {
 }
 
 // Show the side menu
-function menuOpenNav() {
-  document.getElementById("mySidenav").style.width = "250px";
-  document.getElementById("overLay").style.width = "100%";
+function showSideMenu() {
+  document.getElementById("overlay").style.display = "block";
+  document.getElementById("sidenav").style.maxWidth = "250px";
```

Comment:
Can you add a comment that you change maxWidth instead of display here for animation?